### PR TITLE
Add ePBS fork schedule to config

### PR DESCRIPTION
**Pull request by jihoonsong**
**What type of PR is this?**
This PR adds ePBS fork schedule to config. I guess it was omitted as Electra fork epoch is not defined yet, making both `cfg.ElectraForkEpoch` and `cfg.EPBSForkEpoch` have `math.MaxUint64` in `HoleskyConfig` and `SepoliaConfig` after adding ePBS fork. If it ought to be added later, please feel free to close this PR.

### CODE REVIEW

**Review by potuz — APPROVED** [review]
This will conflict with Peer-DAS, but it should still be safe to add.

File: beacon-chain/sync/backfill/verify_test.go [hunk 1]
```diff
@@ -30,8 +30,8 @@ func TestDomainCache(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, len(fsched), len(dc.forkDomains))
 	for i := range fsched {
-		e := fsched[i].Epoch
-		ad, err := dc.forEpoch(e)
+		n := fsched[i].Name
+		ad, err := dc.forName(n)
```
Comment (potuz) [inline beacon-chain/sync/backfill/verify_test.go:34]: Why this change?

Comment (jihoonsong) [reply beacon-chain/sync/backfill/verify_test.go:34]: Using epoch to query fork domain is based an assumption that no two updates have a same epoch. While it's a solid one, it doesn't hold when more than two updates are underway like current case — Electra and ePBS. I think assuming there will be no duplicate name is slightly better than epoch.
